cli/printer/printer_test.go

```diff
@@ -330,18 +330,20 @@ func TestPrinter_Assessment(t *testing.T) {
 				"",
 			}, "\n"),
 		},
-		{
-			"mondoo.build == 1;user(name: 'notthere').authorizedkeys.file",
-			strings.Join([]string{
-				"[failed] mondoo.build == 1;user(name: 'notthere').authorizedkeys.file",
-				"  [failed] mondoo.build == 1",
-				"    expected: == 1",
-				"    actual:   \"development\"",
-				"  [failed] user.authorizedkeys.file",
-				"    error: failed to create resource 'user': user 'notthere' does not exist",
-				"",
-			}, "\n"),
-		},
+		// // TODO: this test needs fixing for the mixed output use-case,
```

Comment:
See https://github.com/mondoohq/cnquery/issues/263 for why this was deactivated and how we can tackle it.